### PR TITLE
Configure nginx for docs and edit vagrant file 

### PR DESCRIPTION
**Pull request by chaimabg**

### CODE REVIEW

File: hivelvet-docs/build-installer-dev.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (GhaziTriki) [inline hivelvet-docs/build-installer-dev.sh:1]: Looks good. I think renaming it to `build.sh` or `build-docs.sh` is better. Using `build-""installer"".sh`, installer is the installer frontend app.

File: hivelvet-docs/run-installer-dev.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (GhaziTriki) [inline hivelvet-docs/run-installer-dev.sh:1]: Since docs are built, I don't it should be served via pm2. Or do we need it for dev?

**Review by GhaziTriki — CHANGES_REQUESTED** [review]

File: hivelvet-docs/build-installer-dev.sh (hunk 1 above)
Comment (chaimabg) [reply hivelvet-docs/build-installer-dev.sh:1]: okay, i will rename it

File: hivelvet-docs/run-installer-dev.sh (hunk 2 above)
Comment (chaimabg) [reply hivelvet-docs/run-installer-dev.sh:1]: In fact, we need to serve the application for the dev (as we did on hivelvet front-end) so I used pm2, the website will be accessed using the url **http://docs.hivelvet.test:3000/**

**Review by GhaziTriki — APPROVED** [review]

**Pull request merged**